content/_guides/helpers.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Advice for Helpers
+category: chanop
+---
+
+Communities are healthiest when everyone works together to maintain the space
+and shape the culture. If we leave all the emotional labor for Operators to
+deal with, they will be more likely to
+[burn out](catalyst#remember-to-take-breaks-to-avoid-burn-out).
+
+These are some things non-Operator helpers can keep in mind during challenging
+situations to avoid making more work for others.
+
+## Let small things slide
+
+The best response to minor annoying behavior is often to ignore it. Drawing
```

Comment:
One method we use a lot in sports, that I wonder is worth mentioning, or if it's too complex to get across well, is to not comment on mistakes the first two-three times since the person is likely to self-reflect on what they did, and on the third/fourth time you mention it in some relative privacy and/or in a very non-confronting/non-accusing way, and ask if they want more detail and or help.

Reply:
Essentially letting small things slide forever might not be the best idea always, if it results in a slow but steady escalation/stretching the limits

Reply:
yeah. i think elsewhere or previously here it says "initially ignore it"